packages/chart/src/hooks/useReduceData.ts

```diff
@@ -2,8 +2,35 @@ function useReduceData(config, data) {
   // for combo charts check if all  Data Series selected to Bar;
   const isBar = config?.series?.every(element => element?.type === 'Bar')
   // for combo charts check if all Data series selected Line or dashed-md/sm/lg.
-  const isAllLine = config?.series?.every(el => el.type === 'Line' || el.type=== 'dashed-sm'|| el.type=== 'dashed-md' || el.type=== 'dashed-lg');
-
+  const isAllLine = config?.series?.every(el => el.type === 'Line' || el.type === 'dashed-sm' || el.type === 'dashed-md' || el.type === 'dashed-lg')
+  const isNumber = value => {
```

Comment:
THis one is slightly different than the other one. It also cleans out some bad characters before the check rather than having to add clean and isNum in the ternary operators below.